udata_front/theme/gouvfr/datagouv-components/src/components/DatasetCard/DatasetCard.stories.ts

```diff
@@ -0,0 +1,159 @@
+    license: false,
+    resources_documentation: true,
+    score: 0.9,
+    spatial: false,
+    temporal_coverage: false,
+    update_frequency: false,
+    update_fulfilled_in_time: false,
+  },
+  metrics: { discussions: 12, followers: 25, reuses: 8, views: 59 },
+  title: "Données changement climatique - SQR",
+  acronym: "DCC",
+  archived: false,
+  description: "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.",
+  tags: null,
+  license: "lov2",
+  frequency: "Unknown",
+  temporal_coverage: "Test",
+  frequency_date: null,
+  private: false,
+  spatial: null,
+  badges: [],
+  owner: null,
+  organization: {
+    id: "some_id",
+    acronym: null,
+    logo: "https://demo-static.data.gouv.fr/avatars/92/6284d057ca4279af35e9b62bc62639-100.png",
+    logo_thumbnail: "https://demo-static.data.gouv.fr/avatars/92/6284d057ca4279af35e9b62bc62639-100.png",
+    page: "https://demo.data.gouv.fr/fr/organizations/test-meteo-france/",
+    badges: [],
+    name: "[TEST] Météo France",
+    slug: "test-meteo-france",
+    uri: "https://demo.data.gouv.fr/api/1/organizations/test-meteo-france/"
+  }
+};
+
+const args = {
+  dataset: dataset,
+  datasetUrl: "/datasets/6571faa17f46a65ee05c4d17",
+};
+
+export const DatasetCardWithDatasetUrl: StoryObj<typeof meta> = {
+  render: (args) => ({
+    components: { DatasetCard },
+    setup() {
+      return { args };
+    },
+    template: `<DatasetCard v-bind="args"/>`,
+  }),
+  args,
+};
+
+export const DatasetCardWithNoQualityScore: StoryObj<typeof meta> = {
+  render: (args) => ({
+    components: { DatasetCard },
+    setup() {
+      return { args };
+    },
+    template: `<DatasetCard v-bind="args" :showQualityScore="false"/>`,
+  }),
+  args,
+};
+
+export const DatasetCardWithDatasetAndOrganizationUrl: StoryObj<typeof meta> = {
+  render: (args) => ({
+    components: { DatasetCard },
+    setup() {
+      return { args };
+    },
+    template: `<DatasetCard v-bind="args"/>`,
+  }),
+  args: {
+    ...args,
+    organizationUrl: "/organizations/another-url-easier-to-distinguish"
+  },
+};
+
+export const RouterDatasetCardWithDatasetUrl: StoryObj<typeof meta> = {
+  render: (args) => ({
+    components: { DatasetCard },
+    setup() {
+      return { args };
+    },
+    template: `<DatasetCard v-bind="args"/><router-view></router-view>`,
+  }),
+  decorators: [vueRouter(datasetRoutes)],
+  args,
+};
+
+export const RouterDatasetCardWithDatasetAndOrganizationUrl: StoryObj<typeof meta> = {
+  render: (args) => ({
+    components: { DatasetCard },
+    setup() {
+      return { args };
+    },
+    template: `<DatasetCard v-bind="args"/><router-view></router-view>`,
+  }),
+  decorators: [vueRouter(datasetRoutes)],
+  args: {
+    ...args,
+    organizationUrl: "/organizations/another-url-easier-to-distinguish"
```

Comment:
This link is not working anymore

Reply:
The link doesn't work when you are already in a subpage. This seems related to how vue-router interact with storybook but I don't think it's really important to fix